scripts/fires/README.md

```diff
@@ -0,0 +1,20 @@
+# Wildland Fire Interagency Geospatial Services (WFIGS)
```

Comment:
please also add details here about the API, the parameters for it, and an example request URL. basically, we need to add enough details here for someone to be able to come in and understand how we did the import.

Reply:
Done